docs/content/mongocxx-v3/installation/_index.md

```diff
@@ -36,6 +36,64 @@ The Mongo C++ driver is available in the following package managers.
 - [Vcpkg](https://vcpkg.io/) (search for mongo-cxx-driver)
 - [Conan](https://conan.io/center/recipes/mongo-cxx-driver)
 
+### Vcpkg Install Instructions
+
+If you do not already have Vcpkg installed, install it with the following
+command:
+```
+$ git clone https://github.com/Microsoft/vcpkg.git
+$ cd vcpkg
+$ ./bootstrap-vcpkg.sh
+```
+
+Optionally, to install with Visual Studio integration:
+```
+vcpkg integrate install
+```
+
+Install the driver.
+```
+$ ./vcpkg install mongo-cxx-driver
+```
+
+You can find the header files in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/include/
+```
+
+The library files are in:
+```
+vcpkg/installed/<CPU ARCHITECTURE>-<OPERATING SYSTEM>/lib/
+```
+
+### Conan Install Instructions
+
+If you do not already have Conan installed, then install it and run the Conan
+initalization command below:
+```
+$ pip install conan
+$ conan profile detect --force
```

Comment:
```suggestion
$ conan profile detect
```

Suggest removing `--force`. May prevent an accidental overwrite of an existing profile.

Reply:
Conan's own instructions include the `--force`

https://docs.conan.io/2/tutorial/consuming_packages/build_simple_cmake_project.html